test/unit/services/githubService.test.js

```diff
@@ -138,4 +138,15 @@ describe("githubService", function () {
       expect(response).to.be.equal(`https://api.github.com/orgs/Real-Dev-Squad/issues`);
     });
   });
+
+  describe("fetchOpenIssues with search query param", function () {
+    it("Should generate the correct url to fetch open issues with search param", async function () {
+      const response = await githubService.fetchOpenIssues({
+        searchString: "website",
+      });
+      expect(response).to.be.equal(
+        "https://api.github.com/search/issues?q=website+org%3AReal-Dev-Squad+type%3Aissue+is%3Aopen+created%3A%3E%3D2023-01-01&sort=created&order=desc&per_page=100&page=1"
```

Comment:
can we use interpolation here and give the URL a proper structure so that it is easy to write tests for something similar in the future?

Reply:
Although this has been the convention which was already followed, we can use the GitHub URL and Org name from the config. Other's are not common to every test. It's common to this particular test.

Reply:
Done.